packages/itwinui-css/src/tile/tile.scss

```diff
@@ -175,9 +196,31 @@
     border-bottom: 1px solid t(iui-color-background-border);
   }
 
+  &.iui-disabled {
+    filter: grayscale(100%);
+    cursor: not-allowed;
+    border-bottom: 4px solid t(iui-color-background-border);
```

Comment:
this should not even work as you are using `t` without `themed` mixin...
Are you sure about border of 4px?
Could use variable for it too (`$iui-xs`)